specification/agrifood/data-plane/Microsoft.AgFoodPlatform/preview/2023-04-01-preview/agfood.json

```diff
@@ -18398,6 +18368,57 @@
         }
       }
     },
+    "AuthCredentials": {
+      "description": "ConnectorCredentials.",
+      "required": [
+        "authType"
+      ],
+      "type": "object",
+      "properties": {
+        "authType": {
+          "$ref": "#/definitions/AuthCredentialsEnum"
+        }
+      }
+    },
+    "AuthCredentialsEnum": {
+      "description": "CredentialTypeEnum.",
+      "enum": [
+        "BasicAuth",
+        "ApiKeyAuth"
```

Comment:
lets add ClientIdClientSecretCredentials as well which is needed for sentinel hub